meilisearch-core/src/update/synonyms_update.rs

```diff
@@ -92,28 +92,11 @@ pub fn apply_synonyms_addition(
         synonyms_store.put_synonyms(writer, word.as_bytes(), &alternatives)?;
```

Comment:
Here you put the alternatives to a word but you did not clear the previous ones.

Reply:
Ok ! Got it, I will fix it !